packages/api/core/src/api/start.ts

```diff
@@ -153,6 +153,8 @@ export default async ({
       } as NodeJS.ProcessEnv,
     };
 
+    delete spawnOpts.env.NODE_OPTIONS;
```

Comment:
Can you explain why this is necessary? I'm a bit weary about what other effects this might have.

Reply:
See below